workflow-templates/im-deploy-az-app-service-manually-to-prod-subn.yml

```diff
@@ -252,7 +270,16 @@ jobs:
           deploy-status: ${{ steps.conclusion.conclusion }}
           timezone: 'america/denver'
 
-      # TODO: Remove if not sending Teams notifications
+      # TODO: Remove the following two steps if not sending Teams notifications
+      # TODO:  If there is only one environment in the prod subscription, the following step can be removed and a single
+      # environment variable can be added at the workflow level above for AZ_FUNC_NAME
+      - name: Set the Azure Function Name
+        uses: im-open/set-variable-based-on-environment@v1.0.0
+        with:
+          variable-name: 'AZ_FUNC_NAME'
```

Comment:
Just want to double check on `AZ_FUNC_NAME` vs. `AZ_APP_NAME`. The next steps is using `AZ_APP_NAME`, so I thought it'd be best to ask.